sqlmesh/integrations/github/cicd/controller.py

```diff
@@ -474,7 +476,7 @@ def conclusion_handler(
                 }[status],
                 None,
             ),
-            conclusion_handler=conclusion_handler,
+            conclusion_handler=functools.partial(conclusion_handler, result=result),
```

Comment:
i never liked partials, always preferred lambda

Reply:
Right tool for the job right? Making a lambda around it is basically creating a partial without using it.